source/index.rst

```diff
@@ -29,6 +29,8 @@ We have a number of guides, starting with a guide to cloudstack's terminology an
 
 Information can also be found at CloudStack's wiki https://cwiki.apache.org/confluence/display/CLOUDSTACK/Home and on cloudstack mailing lists http://cloudstack.apache.org/mailing-lists.html
 
+Documentation is maintained at https://github.com/apache/cloudstack-documentation/
```

Comment:
```suggestion
```

this may not be required here, as documentation oh GitHub details is provided below.